File: GOToPTTK/GOToPTTK-Web/GOToPTTK/Model/Responses/VerifyRequest.cs
using GOToPTTK.Model.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GOToPTTK.Model.Responses
{
    public class VerifyRequest
    {
        public DateTime VerifyDate { get; set; }
        public DateTime SendVerifyDate { get; set; }
        public string Issues { get; set; }
        public int TripId { get; set; }
        public int GuideId { get; set; }
        public int? TripRouteId { get; set; }
        public string VerificationStatusId { get; set; }

        public static VerifyRequest BuildFromModel(Weryfikacja verify)
        {
            VerifyRequest request = new VerifyRequest()
            {
                VerifyDate = verify.DataWeryfikacji,
                SendVerifyDate = verify.DataWyslaniaDoWeryfikacji,
                Issues = verify.Uwagi,
                TripId = verify.WycieczkaId,
                GuideId = verify.PrzodownikUzytkownikId,
                TripRouteId = verify.OdcinekId == null ? -1 : verify.OdcinekId,
                VerificationStatusId = verify.StatusWeryfikacji
            };

            return request;
        }
    }
}

File: GOToPTTK/GOToPTTK-Mobile/app/src/main/java/pl/edu/pwr/gotopttk/Model/Entities/VerifyRequest.java
package pl.edu.pwr.gotopttk.Model.Entities;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.Calendar;
import java.util.Date;

public class VerifyRequest {

    @SerializedName("VerifyDate")
    @Expose
    public Date verifyDate;
    @SerializedName("SendVerifyDate")
    @Expose
    public Date sendVerifyDate;
    @SerializedName("Issues")
    @Expose
    public String issues;
    @SerializedName("GuideId")
    @Expose
    public int guideId;
    @SerializedName("TripId")
    @Expose
    public int tripId;
    @SerializedName("TripRouteId")
    @Expose
    public Integer tripRouteId;
    @SerializedName("VerificationStatusId")
    @Expose
    public String verificationStatusId;

    public VerifyRequest(Date sendDate, String issues, int tripId, int guideId, int trip_route_id, String state){
        verifyDate = Calendar.getInstance().getTime();
        sendVerifyDate = sendDate;
        this.issues = issues;
        this.tripId = tripId;
        this.guideId = guideId;
        this.tripRouteId = trip_route_id;
        this.verificationStatusId = state;
    }
}
